src/module.c

```diff
@@ -876,6 +876,18 @@ int64_t commandFlagsFromString(char *s) {
  *                     to authenticate a client. 
  * * **"may-replicate"**: This command may generate replication traffic, even
  *                        though it's not a write command.  
+ *
+ * The last three parameters specify which arguments of the new command are
+ * Redis keys.
+ *
+ * * 'firstkey': One-based index of the first argument that's a key.
+ *               0 for commands with no keys.
+ * * 'lastkey':  One-based index of the last argument that's a key.
+ *               0 for commands with no keys.
```

Comment:
we should probably mention negative indexes (i.e. -1 is the last argument).
or this whole section may be more easily resolved by a reference to the external documentation for details.